File: Agendor.Core/Entities/Consulta.cs
namespace Agendor.Core.Entities
{
    public class Consulta
    {
        public Guid ConsultaId { get; set; }
        public DateTime DataHora { get; set; }
        public Guid PacienteId { get; set; }
        public Guid MedicoId { get; set; }
        public DateOnly DataDia => DateOnly.FromDateTime(DataHora);
    }
}

File: agendor.client/src/app/interfaces/consulta.ts
export interface ConsultaCreateDto {
  medicoId: string;
  pacienteId: string;
  dataHora: string; // ISO string
}

export interface ConsultaResponseDto {
  consultaId: string;
  medicoId: string;
  pacienteId: string;
  dataHora: string;
  especialidade?: string; 
}
